airflow/configuration.py

```diff
@@ -2011,6 +2011,13 @@ def load_standard_airflow_configuration(airflow_config_parser: AirflowConfigPars
             # there
             AIRFLOW_HOME = airflow_config_parser.get("core", "airflow_home")  # type: ignore[assignment]
             warnings.warn(msg, category=DeprecationWarning)
+    # Set the WEBSERVER_CONFIG variable
+    set_webserver_config(airflow_config_parser)
```

Comment:
Let's just do it here, not sure we really need another function for it.

We should also stop setting it in `write_webserver_configuration_if_needed` as well, and just have that do the writing.

I think we can also undo the test change in https://github.com/apache/airflow/commit/6362ba5ab45a38008814616df4e17717cc3726c3 then as well?

Reply:
Done. Moved the setting to the appropriate place and improved the test